JenkinsfileRelease

```diff
@@ -0,0 +1,80 @@
+pipeline {
+
+  parameters {
+    gitParameter name: 'branch',
+                description: 'The branch to check out',
+                branchFilter: 'origin/(.*)',
+                defaultValue: 'master',
+                selectedValue: 'DEFAULT',
+                type: 'PT_BRANCH',
+                listSize: '0',
+                quickFilterEnabled: true
+
+    string name: 'ReleaseVersion',
+         description: 'Version to release',
+         defaultValue: ''
+  }
+
+  agent {
+    label 'graphdb-jenkins-node'
+  }
+
+  options {
+    disableConcurrentBuilds()
+    timeout(time: 15, unit: 'MINUTES')
+    timestamps()
+  }
+
+  environment {
+    CI = "true"
+    NPM_TOKEN = credentials('npm-token')
+  }
+
+  stages {
+    stage ('Prepare') {
+      steps {
+        // Switch to branch
+        sh "git checkout ${branch}"
+
+        // Change versions
+        sh "npm version --allow-same-version --git-tag-version=false ${ReleaseVersion}"
+        dir("test-cypress/") {
+          sh "npm version --allow-same-version --git-tag-version=false ${ReleaseVersion}"
```

Comment:
`--allow-same-version` do want this?

Reply:
Agree, this was copy/pasted from the platform pipeline, may be @mihailradkov can put some light why do you use it there.

Reply:
we don't publish the platfowm workbench in npm